pkg/scheduler/objects/preemption.go

```diff
@@ -326,7 +326,7 @@ func (p *Preemptor) calculateVictimsByNode(nodeAvailable *resources.Resource, po
 					// removing task does not violate queue constraints, adjust queue and node
 					nodeCurrentAvailable.AddTo(victim.GetAllocatedResource())
 					// check if ask now fits and we haven't had this happen before
-					if nodeCurrentAvailable.FitInMaxUndef(p.ask.GetAllocatedResource()) && index < 0 {
+					if nodeCurrentAvailable.FitIn(p.ask.GetAllocatedResource()) && index < 0 {
 						index = len(results)
 					}
```

Comment:
Can we add a test case for this? I'm worried that such small things can change and we have no corresponding test which tells us if there's something wrong.

Reply:
I am happy to do this without a test. We looked at adding a test but we need to rip apart processing too much to be able to check the return value of the method we are changing. The result of `calculateVictimsByNode` gets hidden too much to be easily accessible. The setup required for testing the function directly is not straight forward and fragile.